scripts/perf_ci.py

```
@@ -0,0 +1,389 @@
+    cmd = [
+        "valgrind",
+        "--tool=callgrind",
+        "--callgrind-out-file=%s" % out_file,
+        "--quiet",
+        binary,
+        # Never write .pyc files. The cache is populated up front by
+        # warm_bytecode_cache; keeping the measured runs read-only means a
+        # workload can neither pay to compile bytecode for a later one nor
+        # race another measurement writing the same file.
+        "-B",
+    ] + argv
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

(RUF005)